harvard-lts/CURIOSity
# Reindexing table

**Add SetSpec to Table**
* * *


# What does this Pull Request do?
Adds the set spec name in to the table.

# How should this be tested?

Spin up this branch -- you should see the set spec name in the history and if you harvest new items you'll see the set spec name for the harvests

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
